documentation/modules/rn-2.6.adoc

```diff
@@ -58,9 +58,25 @@ The {project-short} web console can no longer download logs. With this update, y
 
 This part provides features and enhancements introduced in {project-full} 2.6:
 
+.Preservation of a static IP in a migration from vSphere
+
+You can now preserve a static IP when importing VMs that have vSphere as their source provider. link:https://issuesredhat.com/browse/MTV-1148[(MTV-1148)]
+
+.Entering a list of LUKS decryption passphrases in a vSphere migration plan
+
+You can now enter a single list of multiple Linux Unified Key Setup (LUKS) decryption passphrases when you create a vSphere migration plan, rather than having to enter them one-by-one. For each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device. link:https://issuesredhat.com/browse/MTV-831[(MTV-831)]
```

Comment:
Broken link - 

```suggestion
You can now enter a single list of multiple Linux Unified Key Setup (LUKS) decryption passphrases when you create a vSphere migration plan, rather than having to enter them one-by-one. For each LUKS-encrypted device, {project-short} tries each passphrase until one unlocks the device. link:https://issues.redhat.com/browse/MTV-831[(MTV-831)]
```